parser/parser/cc/grammars/typedruby.ypp

```diff
@@ -927,28 +928,71 @@ int yylex(parser::semantic_type *lval, ruby_parser::location *lloc, ruby_parser:
                       driver.lex.context.allowNumparams = false;
                     }
 
-        defn_head: k_def def_name
+       defn_head: k_def def_name
```

Comment:
Some of these rules were misaligned from all the others by 1 char. The diff looks better ignoring whitespace.